src/main/java/DukeException.java

```diff
@@ -0,0 +1,5 @@
+public class DukeException extends Exception{
```

Comment:
Missing space between `Exception` and `{`

Noticed a few instances of this elsewhere as well